tests/mocks/mock_perf_analyzer.py

```diff
@@ -42,19 +42,26 @@ def __init__(self):
         self.patcher_called_process_error = patch(
             'model_analyzer.perf_analyzer.perf_analyzer.CalledProcessError',
             MockCalledProcessError)
+        super().__init__()
+
+    def start(self):
```

Comment:
Same

Reply:
Yes, I use the patchers that I create to change the behaviour of the mock during the test.